paymentsheet/src/test/java/com/stripe/android/paymentsheet/paymentdatacollection/bacs/BacsMandateConfirmationViewModelTest.kt

```diff
@@ -0,0 +1,86 @@
+package com.stripe.android.paymentsheet.paymentdatacollection.bacs
+
+import app.cash.turbine.test
+import com.google.common.truth.Truth.assertThat
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.test.UnconfinedTestDispatcher
+import kotlinx.coroutines.test.resetMain
+import kotlinx.coroutines.test.runTest
+import kotlinx.coroutines.test.setMain
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+
+class BacsMandateConfirmationViewModelTest {
+    private val viewModel = BacsMandateConfirmationViewModel(
```

Comment:
We should avoid sharing an instance across tests (even if it might be ok in this case). Can you switch to a `createViewModel()`-like method and create a new instance in every test?